next.config.mjs

```diff
@@ -23,4 +23,5 @@ export default withNextra({
   images: { unoptimized: true },
   outputFileTracing: false,
   basePath: process.env.NEXT_BASE_PATH || '',
+  experimental: { mdxRs: true },
```

Comment:
Is this related to swapping to nodevu?

Reply:
Not at all! This should be removed lol. It also has 0 effect as nextra uses its own mdx stuff